showcase-docs/customization.md

```diff
@@ -106,3 +106,31 @@ app:
 
 ![Example Light Mode Sidebar Indicator](images/example-sidebar-indicator-light.png)
 ![Example Dark Mode Sidebar Indicator](images/example-sidebar-indicator-dark.png)
+
+## Customizing Support button content
+
+To customize the Support button content, provide your support team links and icons to the `app.support` field of the `app-config.yaml` file.
+
+Example configurations:
+
+```
+app:
+  support:
+    url: https://github.com/backstage/backstage/issues        # Used as contact support link on common ErrorPage
+    items:                                                    # Used by common SupportButton component
+      - title: Issues
+        icon: github
+        links:
+          - url: https://github.com/backstage/backstage/issues
+            title: GitHub Issues
+      - title: Red Hat Developer Hub
+        icon: chat
+        links:
+          - url: https://discord.gg/backstage-687207715902193673
+            title: '#forum-rhdh'
```

Comment:
```suggestion
      - title: Issues
        icon: github
        links:
          - url: https://github.com/janus-idp/backstage-showcase/issues
            title: GitHub Issues
```

For example let's point to this repo itself.

Reply:
Good catch @invincibleJai . I missed this configure when I set for testing and thanks for the suggestion. I wanted to show two items in the example and added the Janus community link. Let me know how do you think, thanks.
<img width="287" alt="example-support-button" src="https://github.com/janus-idp/backstage-showcase/assets/26255262/ca25aaf4-1f66-4125-969f-5fdbfcb36c98">